docs/source/concepts/prompt_target.rst

```diff
@@ -50,7 +48,7 @@ A prompt target configuration includes the following elements:
 
 - ``name``: A unique identifier for the prompt target.
 - ``description``: A brief explanation of what the prompt target does.
-- ``endpoint``: The API endpoint or function that handles the prompt.
+- ``endpoint`` (Optional): Used for tools scenarios, where a path, http_method, and name are specified.
```

Comment:
I wouldn't say optional. Its only optional when agent_orchestrator is used. Otherwise its required and config check would fail if its missing.